pkg/pillar/types/domainmgrtypes.go

```diff
@@ -24,12 +24,16 @@ type DomainConfig struct {
 	Activate       bool   // Actually start the domU as opposed to prepare
 	AppNum         int    // From networking; makes the name unique
 	VmConfig
-	DiskConfigList    []DiskConfig
-	VifList           []VifInfo
-	IoAdapterList     []IoAdapter
+	DiskConfigList []DiskConfig
+	VifList        []VifInfo
+	IoAdapterList  []IoAdapter
+	// bo be deprecated, use CipherBlock instead
```

Comment:
bo? XXX is useful to make it easier to find things to clean up.

Reply:
sure